storage/innobase/include/row0mysql.h

```diff
@@ -681,6 +681,12 @@ struct mysql_row_templ_t {
 					Innobase record in the current index;
 					not defined if template_type is
 					ROW_MYSQL_WHOLE_ROW */
+	bool	rec_field_is_prefix;	/* is this field in a prefix index? */
```

Comment:
is `rec_field_is_prefix` member needed at all?
can `(rec_prefix_field_no != ULINT_UNDEFINED)` be used as the same indicator?